### ISSUE TITLE
Discussion the tech stack / dev tool stack

### ISSUE THREAD

**Issue by keerthivasan-r** [Kreijstal/diytest#4]
In this thread, we can discuss about the tech stack choice for the website prototype requirements and other tools that would be used as part of development.

**Comment by keerthivasan-r** [comment]
Just for Markdown parsing, in the client - we might make of this tool - [Showdown](http://showdownjs.com/)

**Comment by Kreijstal** [comment]
> Just for Markdown parsing, in the client - we might make of this tool - [Showdown](http://showdownjs.com/)

sure, we would have to see hard it is to add features on this library.

**Comment by keerthivasan-r** [comment]
yeap, right. I have start to take a peep inside the library. Do you suggest anything else?

**Comment by AMacmi01** [comment]
How about making use of the MERN stack? MongoDB, Express, React and Nodejs. Redux can be used to talk to the backend and manage app state. Do you have any other ideas for what you wan to use already aside from Showdown?

**Comment by sedexdev** [comment]
How about making use of the MERN stack? MongoDB, Express, React and Nodejs. Redux can be used to talk to the backend and manage app state. Do you have any other ideas for what you wan to use already aside from Showdown?

**Comment by keerthivasan-r** [comment]
Good, I am completely fine with it. This is definitely a solid approach on this architecture. How do we start? we would have to complete the use cases and then start working towards it

**Comment by sedexdev** [comment]
Could start with an SRS (Software Requirements Specification) and then think about the implementation. Once a document is drawn up then we can see what steps to take first. I'd lik to mention that I'm currently finishing the 3rd year of my degree and am pretty busy so may not be able to dedicate loads of time to this project, but I would like to help when I can with ideas/code

**Comment by pambrocio** [comment]
Is this final

**Comment by pambrocio** [comment]
??

**Comment by sedexdev** [comment]
@keerthivasan-r have you got any further with putting some requirements together?

**Comment by keerthivasan-r** [comment]
hi @AndrewMacmillan87 I am sorry for late reply. I will start putting together the requirements and will share end of this week

**Comment by keerthivasan-r** [comment]
> Is this final

@pambrocio I'm not sure what you are asking here

**Comment by pambrocio** [comment]
I was asking about the technology stack

**Comment by keerthivasan-r** [comment]
@pambrocio Still its in flux only. we should take views from everyone and move forward with contributors. I prefer MEAN stack personally since i have good experience. But, i am open to React as well. I also want to have a single backend - restful api, which we can split later when there is a need to do so.

**Comment by souravghosh97** [comment]
I can give my suggestions
1. UI use react (with other js if needed)
2. Backend Spring MVC
Have you ever thought where you will host this ? 
https://medium.com/@otienojulie/free-resources-to-build-your-next-open-source-project-36d096c536b5

**Comment by keerthivasan-r** [comment]
@souravghosh97  Yep, it is indeed good options to choose. The link you shared is promising. Let us wait for a few more days and then kick start with development. Will you be able to contribute to both UI and backend? Thank you!

**Comment by keerthivasan-r** [comment]
This is something that we have to take decision. Can we go for multiple repo? UI, Backend etc
